screens/dashboard/index.js

```diff
@@ -240,36 +160,9 @@ class Dashboard extends Component<Props> {
           </TouchableHighlight>
         </View>
         <ScrollView style={styles.main}>
-          <View style={styles.padding}>
-            <View style={styles.upcomingBox}>
-              <Text style={[styles.blockTitle, styles.upcomingTitle]}>COMING UP:</Text>
-              {this.showUpcoming('first')}
-              {
-                this.state.expanded1 && (
-                  <View style={styles.dashColumn}>
-                    {this.props.children}
-                    {this.showUpcoming('remaining')}
-                  </View>
-                )
-              }
 
-              <View style={styles.moreButton}>
-                <View style={styles.dashRow}>
-                  <Text style={styles.moreButton}></Text>
-                  <TouchableHighlight
-                    style={styles.dashButton}
-                    onPress={this.toggle1.bind(this)}
-                    underlayColor="transparent">
-                    <Icon
-                      style={[styles.FAIcon, styles.icon1]}
-                      name={icon1}
-                    />
-                  </TouchableHighlight>
-                </View>
-              </View>
+          {/* <Upcoming /> */}
```

Comment:
In general, we should either delete code instead of commenting it out, or leave a comment explaining why the code is temporarily being commented out.  

In this case since Siuan is still working on the design, I think it's reasonable to move this to a component and comment it out, so just add a comment that would help someone else understand why this component is being suppressed, and when it is ok to bring it back or delete the code entirely.